assets/src/edit-story/components/panels/animation/directionRadioInput.js

```diff
@@ -214,11 +281,14 @@ export const DirectionRadioInput = ({ value, directions = [], onChange }) => {
               id={direction}
               type="radio"
               name="direction"
-              value={direction}
+              value={valueForInternalValue(direction)}
               onChange={onChange}
-              checked={value === direction}
+              checked={value === direction || direction.includes(value)}
+            />
+            <DirectionIndicator
+              direction={direction}
+              selected={value === direction || direction.includes(value)}
```

Comment:
`direction?.includes(value)` not sure if we'll ever come across this, but may be nice if an undefined value ever gets thrown in here